connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -148,6 +143,13 @@ message Read {
     // This is only supported by the JDBC data source.
     repeated string predicates = 5;
   }
+
+  // Options for data sources and named table.
+  //
+  // When using for data sources, the context of this map varies based on the
+  // data source format. This options could be empty for valid data source format.
+  // The map key is case insensitive.
+  map<string, string> options = 3;
```

Comment:
I guess just adding the field to `NamedTable` is simpler and doesn't break anything?

Reply:
What's the policy around "breaking changes"?

Reply:
You must not break.

Reply:
I agree that adding the options to named table is probably the better approach given that we have many relations. 

Alternatively simply support both.

Reply:
Sounds good. I probably will go with this non-breaking approach that only `options` to `NamedTable`.

Reply:
Choose to use non-breaking way to change the proto.